logbook/LogBook-IAG.csv

```diff
@@ -0,0 +1,16 @@
+Location,"23°27’22.73""S 46°45’57.94""W",,
+Elevation,1079 m,,
+Inlet elevation,3 m above ground level,,
+Equipment,CRDS Picarro G2301 922-CFADS2200,,
+Measuring,CO2; CH4,,
+Data site came online,,,
+,,,
+Event,"Start Event
+dd/mm/aaaa hh:mm","End Event
+dd/mm/aaaa hh:mm",Flags
+,,,
```

Comment:
We can change to:
Event,Start_Event,End_Event,Flags

Reply:
Done!